x-pack/plugin/esql/qa/server/multi-node/build.gradle

```diff
@@ -1,11 +1,27 @@
 apply plugin: 'elasticsearch.internal-java-rest-test'
+apply plugin: 'elasticsearch.internal-yaml-rest-test'
 
 dependencies {
   javaRestTestImplementation project(xpackModule('esql:qa:testFixtures'))
   javaRestTestImplementation project(xpackModule('esql:qa:server'))
+  yamlRestTestImplementation project(xpackModule('esql:qa:server'))
 }
 
 
 tasks.named('javaRestTest') {
   usesDefaultDistribution()
 }
+
+restResources {
+  restApi {
+    include '_common', 'bulk', 'get', 'indices', 'esql', 'xpack', 'enrich', 'cluster'
+  }
+  restTests {
+    includeXpack 'esql'
+  }
+}
+
+tasks.named('yamlRestTest') {
+  usesDefaultDistribution()
```

Comment:
We should try (when possible) to avoid relying on the default distribution. If it's possible to run this suite with the integ test distrubution with the ESQL module installed (and maybe others) that's ideal. It makes the tests faster and significantly increases the ability to avoid running the test altogether for changesets that don't affect the tests.

Reply:
I'd love to! I think we should spend some time reworking these tests to try and squeeze them tighter. We really don't need these to be in a `multi-node` and `single-node` package. It's an artifact of the old way but it's a pain to change now.

Reply:
Yeah, having them in separate projects isn't a huge deal. Using the default distro however almost guarantees these tests will never be cached except for minor things like docs changes.

Reply:
I filed https://github.com/elastic/elasticsearch/issues/104558 to track this because I'm not going to do it now. Especially if I want to backport this.